server/odc-server/src/main/java/com/oceanbase/odc/server/web/controller/v2/FlowInstanceController.java

```diff
@@ -105,6 +105,7 @@ public PaginatedResponse<FlowInstanceDetailResp> listFlowInstances(
             @RequestParam(required = false, name = "endTime") Date endTime,
             @RequestParam(name = "createdByCurrentUser") Boolean createdByCurrentUser,
             @RequestParam(name = "approveByCurrentUser") Boolean approveByCurrentUser,
+            @RequestParam(required = false, name = "containsAll", defaultValue = "false") Boolean containsAll,
```

Comment:
if createdByCurrentUser/approveByCurrentUser not set, will it be contains All?

all query parameters can mapping to a filter, if not parameters, means there is not filter ?

Reply:
Currently, the parameter `createdByCurrentUser/approveByCurrentUser` is required, so they will always be set. If `containsAll = true`, then the `containsAll`'s priority is higher than `createdByCurrentUser/approveByCurrentUser`'s; if `containsAll = false`, then the query depends on `createdByCurrentUser/approveByCurrentUser`.
But it seems a bit tricky...